spec/controllers/proposals_controller_spec.rb

```diff
@@ -91,16 +91,22 @@
       end
     end
 
-    context 'details beta' do
+  end
+
+  describe '#show + new details' do
+    before do
```

Comment:
@rememberlenny you going to move this? fine if you'd rather not, just don't want to merge if you are still working on the code